### PR TITLE
feat: Print format in stdout

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Print format information for model when saving/exporting.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes https://github.com/caicloud/ormb/issues/13

Reference to #

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/types.go [hunk 1]
```diff
@@ -16,7 +16,8 @@ type Metadata struct {
 	Description     string            `json:"description,omitempty"`
 	Tags            []string          `json:"tags,omitempty"`
 	Labels          map[string]string `json:"labels,omitempty"`
-	Format          string            `json:"format"`
+	Format          string            `json:"format,omitempty"`
+	Framework       string            `json:"framework,omitempty"`
 	Metrics         []Metric          `json:"metrics,omitempty"`
 	Hyperparameters []Hyperparameter  `json:"hyperparameters,omitempty"`
```
Comment (simon-cj) [inline pkg/model/types.go:22]: In [spec definition](https://github.com/caicloud/ormb/commit/734593e4f5c3d26b0f9d6de55986735a49242aa9\#diff-5441b3fdbf500dd8b6f82d339f005276R88),  I see some field is `REQUIRED`, eg: `format` field , these field SHOULD limit by `json:"format,omitempty" required: "true"`.

Comment (gaocegege) [reply pkg/model/types.go:22]: required: "true" is it really work? Who will check the tag to verify it is a required field?

Comment (simon-cj) [reply pkg/model/types.go:22]: Forget it...... it guess json.Unmarshal will check it, but it is not.

Find other method to verify it in the future.

Comment (gaocegege) [reply pkg/model/types.go:22]: Yeah, I think so. We should have something like https://github.com/jessevdk/go-flags to support the feature.

**Pull request merged**